src/effects/How to make an effect stateless.txt

```diff
@@ -0,0 +1,189 @@
+4 STEPS TO MAKE AN EFFECT STATELESS
+
+1) "Define EffectEchoSettings"
```

Comment:
Of course this guide should say "Let use use EffectEcho as an example" and it might refer to commit ids that made this example transformation.